cmd/pinniped-proxy/src/main.rs

```diff
@@ -26,18 +26,15 @@ mod tls_config;
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    pretty_env_logger::formatted_timed_builder()
-        .format(|buf, record| {
-            writeln!(
-                buf,
-                "{} [{}] - {}",
-                Local::now().format("%Y-%m-%dT%H:%M:%S%.6f"),
-                record.level(),
-                record.args()
-            )
-        })
-        .filter(None, LevelFilter::Info)
-        .init();
+    env_logger::Builder::from_default_env().format(|buf, record| {
```

Comment:
The key change here (in addition to not needing the extra `pretty_env_logger`), is that we're using `from_default_env()` so the `RUST_LOG` env var is actually used (rather than hard-coding the level above).